### PR TITLE
Hide title in frontmatter

### PR DESCRIPTION
**Pull request by archiewood**
### Description

- Addresses #780
- Updates the docs

### Checklist

- [ ] ~~For UI or styling changes, I have added a screenshot or gif showing before & after~~
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### CODE REVIEW

File: packages/preprocess/src/frontmatter/parse-frontmatter.spec.cjs [hunk 1]
```diff
@@ -9,7 +9,8 @@ const {
 	complexFrontmatter,
 	extraFrontmatter,
 	frontmatterWithMarkdownTable,
-	markdownTableWithoutFrontmatter
+	markdownTableWithoutFrontmatter,
+	frontmatterWithTitle
```
Comment (ItsMeBrianD) [inline packages/preprocess/src/frontmatter/parse-frontmatter.spec.cjs:13]: `frontmatterWithTitleHidden` needs to be imported here I believe

**Review by ItsMeBrianD — COMMENTED** [review]
Sorry for leaving the 3 handle opengraph files 😅

This looks good to me though; just one little thing w/ tests

File: packages/preprocess/src/frontmatter/parse-frontmatter.spec.cjs (hunk 1 above)
Comment (archiewood) [reply packages/preprocess/src/frontmatter/parse-frontmatter.spec.cjs]: I was about to ask
Should I remove the redundant og files?

**Review by ItsMeBrianD — APPROVED** [review]

**Comment by archiewood** [comment]
I was about to ask. Should I remove the redundant og files?

I think the handle-og.snippet.svelte and handle-opengraph.cjs do nothing?

**Pull request merged**